packages/odyssey-react/src/components/Select/Select.test.tsx

```diff
@@ -131,5 +158,18 @@ describe("Select", () => {
     />;
   });
 
+  it("Composer can use choices ref from dom", async () => {
+    const { getByText } = render(
+      <RefTree label={label} name={name} children={<div />} />
+    );
+
+    const listbox = screen.getAllByRole(listboxRole)[0];
+    fireEvent.mouseDown(listbox);
+
+    await waitFor(() => {
+      return getByText("Choices!");
+    });
+  });
```

Comment:
the test setup here is a bit counter-intuitive without any jest assertions in the test body itself but I do see what you're doing. Would love to clean this up but won't block the the PR over it. In the future perhaps we can avoid structuring the test only to fail from the polling timeout expiring via `waitFor` and make things more explicit and readable